### PR TITLE
docs: add rrf docs and update reranking notebook with Jina reranker results

### PR DESCRIPTION
**Pull request by AyushExel**
- RRF reranker
- Jina Reranker results

### CODE REVIEW

File: docs/src/reranking/rrf.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+data = [
+    {"text": "hello world"},
+    {"text": "goodbye world"}
+    ]
+tbl = db.create_table("test", schema=Schema, mode="overwrite")
+tbl.add(data)
+reranker = RRFReranker()
+
+# Run hybrid search with a reranker
+tbl.create_fts_index("text", replace=True)
```
Comment (westonpace) [inline docs/src/reranking/rrf.md:33]: Does it matter if you have a vector index?

File: docs/src/reranking/rrf.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+result = tbl.search("hello", query_type="hybrid").rerank(reranker=reranker).to_list()
+
+```
+
+Accepted Arguments
+----------------
+| Argument | Type | Default | Description |
+| --- | --- | --- | --- |
+| `K` | `int` | `60` | A constant used in the RRF formula (default is 60). Experiments indicate that k = 60 was near-optimal, but that the choice is not critical |
+| `return_score` | str | `"relevance"` | Options are "relevance" or "all". The type of score to return. If "relevance", will return only the `_relevance_score. If "all", will return all scores from the vector and FTS search along with the relevance score. |
```
Comment (westonpace) [inline docs/src/reranking/rrf.md:43]:
```suggestion
| `return_score` | str | `"relevance"` | Options are "relevance" or "all". The type of score to return. If "relevance", will return only the `_relevance_score`. If "all", will return all scores from the vector and FTS search along with the relevance score. |
```

File: docs/src/reranking/rrf.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+## Supported Scores for each query type
+You can specify the type of scores you want the reranker to return. The following are the supported scores for each query type:
+
+### Hybrid Search
+|`return_score`| Status | Description |
+| --- | --- | --- |
+| `relevance` | ✅ Supported | Returns only have the `_relevance_score` column |
+| `all` | ✅ Supported | Returns have vector(`_distance`) and FTS(`score`) along with Hybrid Search score(`_distance`) |
```
Comment (westonpace) [inline docs/src/reranking/rrf.md:53]:
```suggestion
| `relevance` | ✅ Supported | Returned rows only have the `_relevance_score` column |
| `all` | ✅ Supported | Returned rows have vector(`_distance`) and FTS(`score`) along with Hybrid Search score(`_relevance_score`) |
```

**Review by westonpace — APPROVED** [review]
Some minor suggestions

File: docs/src/reranking/rrf.md (hunk 1 above)
Comment (AyushExel) [reply docs/src/reranking/rrf.md:33]: a vector index is not required but it'll be fine if it exists. But fts requires fts_index

**Comment by AyushExel** [comment]
Thankyou!

**Pull request merged**